integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -320,6 +322,37 @@ public void testAdminT3ChannelWithJMS(Domain domain) throws Exception {
     logger.info("Done - testAdminT3ChannelWithJMS");
   }
 
+  /**
+   * Verify Load Balancing by deploying and invoking webservicebapp
+   *
+   * @throws Exception
```

Comment:
Should explain when/why exception is thrown. First sentence of javadoc should end with a period.
Please see https://www.oracle.com/technetwork/java/javase/tech/index-137868.html